src/app/shared/components/project-title/project-title.component.ts

```diff
@@ -96,7 +81,7 @@ export class ProjectTitleComponent implements OnInit {
       );
     } else {
       this.errorEmitter.emit(true);
-      this.alertService.error('Invalid user id in URL');
+      this.alertService.error('There is an invalid user ID in the URL');
```

Comment:
Not sure how this applies but userID is no longer required - the entire idea is that if there is no userID we authenticate via LFID (as all other products do) and then you either already have that user UUID or you call `/v4/user-from-token` that will return user data for you.

Reply:
I will create a new PR once this gets merged. This PR aims to move all pages under the LF login.

Reply:
OK.